web/help/rlt-notes-unit1.md

```diff
@@ -0,0 +1,14 @@
+If you do not leave a space before and after the & the popup does not show
+If the student makes change and clicks an example they get a warning. It might be beneficial to mention something about this and/or saving.
```

Comment:
I do think it might help to address some of these UI issues when they come up, yes.  I'd keep it in callout boxes so it doesn't interrupt the flow of the main exposition.